### PR TITLE
Add probablepeople recipe

### PR DESCRIPTION
**Pull request by proinsias**
- [x] Add `doublemetaphone` to `conda-forge`
- [x] Tidy up `meta.yaml` after verifying build requirements

### CODE REVIEW

File: recipes/probablepeople/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - probablepeople
+
+about:
+  home: https://parserator.datamade.us/probablepeople/
+  license: MIT
+  license_family: MIT
+  #license_file: LICENSE # When added to MANIFEST.in – datamade/probablepeople#33
```
Comment (jakirkham) [inline recipes/probablepeople/meta.yaml]: xref: https://github.com/datamade/probablepeople/pull/33

File: recipes/probablepeople/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - python-crfsuite >=0.8
+    - probableparsing
+    - future
+    - doublemetaphone
```
Comment (jakirkham) [inline recipes/probablepeople/meta.yaml]: Please drop the dependencies not needed at build.

**Comment by jakirkham** [comment]
Could you please address the extra build dependencies @proinsias? Then will happily merge.

File: recipes/probablepeople/meta.yaml (hunk 2 above)
Comment (proinsias) [inline recipes/probablepeople/meta.yaml]: These are specified in `setup.py`. I'm now verifying whether they are needed or not.

**Comment by proinsias** [comment]
@jakirkham – I think this is ready now.

File: recipes/probablepeople/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - probablepeople
+
+about:
+  home: https://parserator.datamade.us/probablepeople
+  license: MIT
+  license_family: MIT
+  #license_file: LICENSE # When added to MANIFEST.in – datamade/probablepeople#33
```
Comment (jakirkham) [inline recipes/probablepeople/meta.yaml]: xref: https://github.com/datamade/probablepeople/pull/33

**Comment by jakirkham** [comment]
Thanks @proinsias .

**Pull request merged**